configure_bazel.sh

```diff
@@ -36,6 +36,7 @@ TF_CXX_ABI=( $(python -c 'import tensorflow as tf; print(" ".join(str(tf.__cxx11
 
 write_action_env_to_bazelrc "TF_HEADER_DIR" ${TF_CFLAGS:2}
 write_action_env_to_bazelrc "TF_SHARED_LIBRARY_DIR" ${TF_LFLAGS:2}
+write_to_bazelrc "build --define=grpc_no_ares=true"
```

Comment:
eg, what does this flag do

Reply:
This has to do with how we re pulling in TF code and building it. When we invoke `tf_workspace(...)` I needed to add this to get the tf workspace setup going.